setup.md

```diff
@@ -8,72 +8,156 @@ permalink: /setup/
 This guide describes how to set up your development environment to run Flutter apps on
 iOS or Android.
 
-* TOC Placeholder
-{:toc}
+New to GitHub? First [install the software](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git){: alert-link}
```

Comment:
This new GitHub section seems like a potential distraction; there is a lot of text to read through on those links.

On the two current operating systems we support git should be available, and the user really doesn't need to know how git works just for installation purposes, so I think we can omit it.